app/serializers/search/simple_user_serializer.rb

```diff
@@ -1,6 +1,5 @@
 module Search
-  # to be renamed when ES is gone
-  class PostgresUserSerializer < ApplicationSerializer
+  class SimpleUserSerializer < ApplicationSerializer
```

Comment:
@rhymes any idea what we should name this? We have a few variations of a "user" serializer 🤔